utils/utils.go

```diff
@@ -1,7 +1,7 @@
 // (c) 2019-2024, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
-package atomic
+package utils
```

Comment:
Should we keep this in a separate package like `plugin/evm/client`? IMO we can move `plugin/evm/client.go` to that new package (as we need somewhere to put `NewCChainClient` that doesn't import `plugin/evm` soon)

Reply:
that `utils` is already in a different pkg, just not in the same one with client. Do you want to move it to the same client pkg? Also the new `client` package does not import `plugin/evm` (just `plugin/evm/atomic`).

Reply:
The main concern is that the `utils` package may end up importing more stuff than we want, for example it's already importing avalanchego package, which can cause circular issues again (as the clients are using this function eg in tests)

Reply:
It turned out that we don't actually need those methods in there. Exported those from avalanchego